yarn.lock

```diff
@@ -2819,6 +2819,30 @@
     pouchdb-promise "^6.0.4"
     through2 "^2.0.0"
 
+"@budibase/pro@*":
```

Comment:
I assume a pro change is not needed? You may need to checkout master for the pro module. 

```
cd packages/pro
git pull master
cd ../..
```